book/src/bridge/attributes.md

```diff
@@ -43,3 +43,8 @@ For [`#[qproperty]`](./extern_rustqt.md#properties), a CXX or Rust name can be p
 The `#[auto_cxx_name]` and `#[auto_rust_name]` attributes can be used to automatically rename cxx and rust names.
 These are placed at a block level on `extern "RustQt"` or `extern "C++Qt"` blocks, and will automatically case convert the items inside, unless they specify either a `rust_name` or `cxx_name`.
 By default `#[auto_cxx_name]` will generate a camelCase conversion for`cxx_name` and `#[auto_rust_name]` will generate a snake_case conversion for `rust_name`.
+
+### Automatic wrapping
```

Comment:
an example that is compiled would be good too. I think the load_color here could be changed to use it https://github.com/KDAB/cxx-qt/blob/main/examples/qml_features/rust/src/invokables.rs#L99

So that you have load_color auto_wrapped to read the inner as_qcolor on the Rust type (which could be renamed)